src/main/kotlin/io/reactivex/rxkotlin/Observables.kt

```diff
@@ -141,8 +154,49 @@ inline fun <T, T1, T2, T3, T4, R> Observable<T>.withLatestFrom(o1: ObservableSou
         = withLatestFrom(o1, o2, o3, o4, Function5<T, T1, T2, T3, T4, R> { t, t1, t2, t3, t4 -> combiner.invoke(t, t1, t2, t3, t4) })
 
 /**
+ * Shortcut for [Observable.combineLatest] with two args
```

Comment:
I propose removing `combineLatestWith()` since this is not natively available in RxJava.